converter/src/main/java/gov/cms/qpp/conversion/encode/ClinicalDocumentEncoder.java

```diff
@@ -91,7 +90,7 @@ private JsonWrapper encodeMeasurementSets(Map<TemplateId, Node> childMapByTempla
 		JsonOutputEncoder sectionEncoder;
 
 		for (Node child : childMapByTemplateId.values()) {
-			if (TemplateId.NPI_TIN_ID != child.getType()) {
+//			if (TemplateId.NPI_TIN_ID != child.getType()) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)